### ISSUE TITLE
Tooltip is not accessible using keyboard in the dropdown

### ISSUE THREAD

**Issue by vaishnavi-115** [highcharts/highcharts#17750]
Expected behaviour
Tooltip should be accessible using keyboard keys in dropdown

Actual behaviour
Tooltip is accessible only while using mouse in dropdown

#### Live demo with steps to reproduce

Product version
@fluentui/react": "^8.96.0",

Affected browser(s)
Chrome, Edge

**Comment by pawellysy** [comment]
Hi @vaishnavi-115, thanks for creating this ticket!
You didn't give us enough information to investigate this issue 😅 
Could you please share some more details, preferably with the demo, where the issue can be reproduced? Thanks!

**Comment by vaishnavi-115** [comment]
Hi, I realize that this issue is related to fluent UI, I shall create a bug there.
Thanks!
